py2plantuml.sh

```diff
@@ -3,11 +3,43 @@ from_dir=$1
 out_dir=$2
 plantuml=/opt/homebrew/bin/plantuml
 python=python3
-if [[ ! -d $2 ]]; then
+
+function usage(){
+    echo "$(basename $0) [-i | --from_dir ] [ -o | --out_dir] [-d | --dependency_install] [-h | --help]"
+}
+
+while [[ "$1" != "" ]]; do
+    case $1 in
+        -i | --from_dir )
+        from_dir=$2;
+        shift;
+        ;;
+        -o | --out_dir )
+        out_dir=$2;
+        shift;
+        ;;
+        -d | --dependency_install )
+        echo "Downloading plantuml java binary in the current dir from github......"
```

Comment:
Excellent idea @MSarfraz-CR . 
Do not we need to:
1. Check java exists first?
2. Graphviz needs to be installed as well (https://plantuml.com/de/graphviz-dot)